services/src/main/java/org/keycloak/forms/account/freemarker/model/AuthorizationBean.java

```diff
@@ -234,7 +236,17 @@ public ResourceBean(Resource resource) {
             RealmModel realm = authorization.getRealm();
             resourceServer = new ResourceServerBean(realm.getClientById(resource.getResourceServer().getId()));
             this.resource = resource;
-            owner = authorization.getKeycloakSession().users().getUserById(resource.getOwner(), realm);
+            userOwner = authorization.getKeycloakSession().users().getUserById(resource.getOwner(), realm);
+            if (userOwner == null) {
+                clientOwner = authorization.getKeycloakSession().clientStorageManager().getClientById(resource.getOwner(), realm);
+                ownerName = clientOwner.getClientId();
+            } else if (userOwner.getEmail() != null) {
+                clientOwner = null;
```

Comment:
Do you really need to set this to null?

Reply:
If `clientOwner` has to be final, yes. But since that should be unnecessary, I'll remove the keyword.